content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -2,13 +2,21 @@
 linkTitle: PhysX Assets
 title: Process PhysX Collider Assets
 description: Learn to customize PhysX collider asset processing in Open 3D Engine (O3DE) with Scene Settings.
-weight: 100
+weight: 400
 toc: true
 ---
 
-**Open 3D Engine (O3DE)** has a robust set of options for generating PhysX collider assets. In **Scene Settings**, you can customize PhysX collider asset generation. The generated PhysX colliders are stored in `.pxmesh` product assets. You can add the PhysX collider assets to a **PhysX Mesh Collider** component.
+**Open 3D Engine (O3DE)** has a robust set of options for generating PhysX collider assets. There are three PhysX collider types that you can use in different simulation scenarios. This topic explains the benefits and limitations of the different collider types, as well as the basics of generating PhysX asset colliders with  [**Scene Settings**](/docs/user-guide/assets/scene-settings/scene-settings).
 
-There are three PhysX asset collider types that you can use in different simulation scenarios. This topic explains the benefits and limitations of the different collider types, as well as the basics of generating PhysX asset colliders with Scene Settings.
+The generated PhysX colliders are stored in `.pxmesh` product assets. You can add the PhysX collider assets to a **PhysX Collider** component by selecting **PhysicsAsset** in the component's **Shape** property.
+
+{{< note >}}
+Understanding the [best practices](/docs/user-guide/assets/scene-settings/source-asset-best-practices#physx) for creating PhysX collider source assets can mitigate issues you might encounter when processing colliders for O3DE. For technical details about the data supported by colliders, refer to the [supported 3D scene data](/docs/user-guide/assets/scene-settings/scene-format-support#supported-3d-scene-data) table.
+{{< /note >}}
+
+| O3DE Experience | Time to Complete | Feature Focus | Last Updated |
+| - | - | - | - |
+| Beginner | 25 Minutes | Customized processing of PhysX collider assets from `.fbx` files with Scene Settings. | January 4, 2023 |
 
 ## Entity behavior

```

Comment:
I think there should be some changes introduced by this [PR](https://github.com/o3de/o3de.org/pull/2230/files#diff-687a70c2c47e65118bd6661d32ab408798602293203e4400e42055ce77eb83f9) to development made last month. The changes introduced spread throughout the file. See the [blame](https://github.com/o3de/o3de.org/blame/development/content/docs/learning-guide/tutorials/assets/physx-colliders.md#L17-L19) for this file's history.

Reply:
Adding the correct changes.